config/environments/production.rb

```diff
@@ -16,12 +16,13 @@
 
```

Comment:
Yes, like the new version is using <code>serve_static_files</code> instead of <code>serve_static_assets</code>